adoc/MAIN.release-notes.adoc

```diff
@@ -257,20 +250,20 @@ If you have not applied 4.0.1, the gangway configurations will be reverted to th
 
 The gangway image that shipped with {productname} 4.0 must be updated manually by performing the following steps:
 
-* Delete the gangway deployment completely
+* Download the gangway deployment
 +
 ====
-kubectl -f delete addons/gangway/gangway.yaml
+kubectl get deployment oidc-gangway -oyaml > gangway-deployment.yaml
 ====
-* Delete the original image from node where gangway is running
+* Edit the [path]`gangway-deployment.yaml`, change the gangway image tag from `3.1.0` to `3.1.0-rev4`
 +
```

Comment:
if we delete the stop process and delete image, when we pull should we get the latest build? both 3.1.0 and 3.1.0-rev4 currenly have same image id 8fbb7505645b